GVFS/GVFS.Common/Git/GitProcess.cs

```diff
@@ -674,15 +685,16 @@ private Result InvokeGitOutsideEnlistment(string command)
         /// Invokes git.exe against an enlistment's .git folder.
         /// This method should be used only with git-commands that ignore the working directory
         /// </summary>
-        private Result InvokeGitAgainstDotGitFolder(string command)
+        private Result InvokeGitAgainstDotGitFolder(string command, string gitObjectsDirectory = null)

```

Comment:
It looks like you aren't using `gitObjectsDirectory` here?

Reply:
Yup I found that out also through a new functional test :-).  Yay functional tests!